src/ui/screens/Democracy/DemocracyScreen.tsx

```diff
@@ -261,6 +264,7 @@ const styles = StyleSheet.create({
   },
   standardPadding: {
     padding: standardPadding,
+    marginHorizontal: standardPadding,
```

Comment:
And removing this.

```suggestion
```

Reply:
We need this so that we can have the margin on both sides else the card will be stuck to the end of the screen like this 
<img width="568" alt="image" src="https://user-images.githubusercontent.com/22363352/182844019-ceffe1eb-521e-423c-8b97-d3815520ecc4.png">

Reply:
In that case change the name `standardPadding` to `itemPadding` or something like that.

Reply:
Renamed to `teaserCard` since I use it in both Referendums and Proposals teasers